src/components/Timer.js

```diff
@@ -0,0 +1,55 @@
+import React, { useState, useEffect } from 'react';
+import '../styles/Timer.css'
+
+function Timer() {
+    const [currentValue, setCurrentValue] = useState(0);
+    const [intervalId, setIntervalId] = useState(false);
+
+    const resetTimer = () => {
+        clearInterval(intervalId);
+        setIntervalId(false);
+        setCurrentValue(0);
+    }
+
+    const startTimer = () => {
+        if (!intervalId) {
+            const startTime = Date.now();
+            const interval = setInterval(() => {
+                const value = (currentValue + Math.round((Date.now() - startTime) / 100) / 10);
```

Comment:
Isn't the value you want stored in `currentValue` always `(Date.now() - startTime)/1000`? This looks like the timer will go faster the longer it has been running.

Reply:
The `/10` step was simply to round to the nearest tens digit. No longer relevant with the formatting change.

Reply:
Ok I see the issue. When the "interval" function is set, it gets values once up front from the broader scope and does not get them again. That's why it's not accelerating in use, but why it looks like it should.
To make it more readable, I created a pass-through value that makes it obvious we're only getting that state variable once